libsolidity/interface/SourceReferenceFormatter.cpp

```diff
@@ -79,8 +79,7 @@ void SourceReferenceFormatter::printSourceLocation(SourceLocation const* _locati
 			scanner.lineAtPosition(_location->start) <<
 			endl <<
 			string(startColumn, ' ') <<
-			"^\n" <<
```

Comment:
Weird we haven't used `endl` here :)

Reply:
As far as I know, `endl` is exactly `\n` plus "flush the stream", so in most cases, `\n` would actually be better.

Reply:
Yep, but we consistently use `endl` :)

Reply:
@axix @chriseth 

I updated to use endl in this part of the file.  I did not search through the rest.  You can see this reflected in second commit to the PR.